hs_file_types/models/reftimeseries.py

```diff
@@ -983,13 +989,15 @@ def _validate_json_data(series_data):
                 raise Exception(err_msg.format("Invalid web service URL found"))
 
         if 'method' in series:
-            url = Request(series['method']['methodLink'])
-            if url not in urls:
-                urls.append(url)
-                try:
-                    urlopen(url)
-                except URLError:
-                    raise Exception(err_msg.format("Invalid method link found"))
+            url = series['method']['methodLink'].strip()
+            if url and url.lower() != 'unknown':
+                url = Request(url)
+                if url not in urls:
+                    urls.append(url)
+                    try:
+                        urlopen(url)
```

Comment:
Should this happen before adding url to urls?  I guess it does raise an exception... I assume it just invalidates the whole file so it doesn't matter?

Reply:
We are not tracking only the valid urls. So it doesn't matter.